crawl-ref/source/abyss.cc

```diff
@@ -1580,19 +1577,12 @@ void generate_abyss()
     check_map_validity();
 
     // If we're starting out in the Abyss, make sure the starting grid is
-    // an altar to Lugonu and there's an exit near-by.
+    // an exit.
     // Otherwise, we start out on floor and there's a chance there's an
     // altar near-by.
     if (player_in_starting_abyss())
     {
-        env.grid(ABYSS_CENTRE) = DNGN_ALTAR_LUGONU;
-        const coord_def eloc = _place_feature_near(ABYSS_CENTRE, LOS_RADIUS + 2,
-                                                   DNGN_FLOOR, DNGN_EXIT_ABYSS,
-                                                   50, true);
-        // Now make sure there is a path from the abyss centre to the exit.
-        // If for some reason an exit could not be placed, don't bother.
-        if (eloc == INVALID_COORD || !_abyss_has_path(eloc))
```

Comment:
If you're taking this approach I think you should also delete the body of _abyss_has_path() above, which would now be unused?